### PR TITLE
feat: import strongs definitions

### LINKED ISSUE #165: process to import Strongs Concordance data
* Use the `<strongs_defs>` tag from [this data source](https://raw.githubusercontent.com/morphgnt/strongs-dictionary-xml/master/strongsgreek.xml) to generate lexicon entries.
* We need a script to process this data and update the database so it can be repeated.
* The strongs definition should just be raw text

### PR DESCRIPTION
**Pull request by Pertempto**
## What has changed

Added script to insert strongs definitions into the database. Updated seed dump file.

## Connected Issues

Closes #165

## QA Steps

1. Run `nx run db:prisma migrate reset`
2. In your SQL database, `SELECT * FROM public."LemmaResource" WHERE "resourceCode" = 'STRONGS';`
     - The strongs entries should be shown

## Post-Deployment

- [ ] Add strongs data to the production DB

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
     "@lucia-auth/adapter-prisma": "2.0.0",
     "@prisma/client": "4.11.0",
     "@swc/helpers": "0.5.0",
+    "fast-xml-parser": "4.3.2",
```
Comment (Pertempto) [inline package.json]: This time you don't have to remind me to lock down the version number 😅

**Comment by Pertempto** [comment]
That file only contains the greek strongs definitions. We'll need another source for hebrew strongs definitions.

**Comment by arrocke** [comment]
Can you investigate data whether this can be used instead? https://github.com/openscriptures/strongs/tree/master

**Comment by Pertempto** [comment]
I decided to use the dictionary JS files, because it didn't require any extra parsing or clean up.

**Review by arrocke — APPROVED** [review]

**Pull request merged**